modules/ROOT/partials/configuration/toolbar_mode.adoc

```diff
@@ -36,21 +36,24 @@ The toolbar modes are not available when using link:{baseurl}/configure/editor-a
 
 include::partial$misc/differs_for_mobile.adoc[]
 
+[discrete]
 === Settings
 
 The toolbar mode is specified in the `tinymce.init`. There are four toolbar modes:
 
-* <<floating,Floating>>
-* <<sliding,Sliding>>
-* <<scrolling,Scrolling>>
-* <<wrap,Wrap>>
+* xref:floating[Floating]
+* xref:sliding[Sliding]
+* xref:scrolling[Scrolling]
+* xref:wrap[Wrap]
 
+[[floating]]
 ==== Floating
 
-If the `toolbar_mode` is configured to `floating`, the toolbar appears under the toolbar overflow icon in a floating shelf format when the toolbar overflow icon image:{baseurl}/images/icons/more-drawer.svg[Drawer] is clicked.
+If the `toolbar_mode` is configured to `floating`, the toolbar appears under the toolbar overflow icon in a floating shelf format when the toolbar overflow icon image:icons/more-drawer.svg[Drawer] is clicked.
 
-image::{baseurl}/images/floating_toolbar.png[Floating Toolbar]
+image:floating_toolbar.png[Floating Toolbar]
```

Comment:
This change is incorrect as it's meant to be a block image, not an inline image.
```suggestion
image::floating_toolbar.png[Floating Toolbar]
```